tezos/context/src/working_tree/storage.rs

```diff
@@ -1100,9 +1236,13 @@ impl Storage {
         match inode {
             Inode::Pointers { pointers, .. } => {
                 for pointer in pointers.iter().filter_map(|p| p.as_ref()) {
-                    let inode_id = pointer.inode_id();
-                    let inode = self.get_inode(inode_id)?;
-                    self.iter_inodes_recursive_unsorted(inode, fun)?;
+                    // When the inode is not deserialized, ignore it
```

Comment:
@sebastiencs when is this version that skips non-deserialized inodes used? when committing?

Reply:
Yes on commit, to serialize and hash them